src/govuk/components/back-link/_index.scss

```diff
@@ -35,14 +42,15 @@
     @if $govuk-use-legacy-font {
       // Begin adjustments for font baseline offset
       // These should be removed when legacy font support is dropped
-      top: -1px;
-      bottom: 1px;
+      $offset: govuk-em(1px, $govuk-root-font-size);
+      top: $offset * -1;
+      bottom: $offset;
     } @else {
       top: 0;
```

Comment:
@querkmachine You've got me looking at this more now 🙈 

Do you think we should let browsers calculate the vertical position instead of assuming `top: 0`?

For example, applying the old 50% vertical offset trick puts the chevron about 1px lower on mobile. I've added an example from macOS Safari 16.1 with 2x `window.devicePixelRatio`—looks better aligned?

1st animation frame — original `top: 0`
2nd animation frame — 50% offset (suggestion)

![Back link animation showing 1px vertical shift](https://user-images.githubusercontent.com/415517/201359129-8b63e65b-bdb1-47ec-9eca-e03b5724f310.gif)

```suggestion
      top: 50%;
      margin-top: $chevron-size / -2
```

Reply:
This might be a question for the designers, but I _think_ the intent is that the chevron is aligned with the text baseline, rather than being exactly vertically centred?

Reply:
Yeah best check, Percy shows it well?

As in, small screens are now (correctly) showing a smaller arrow but it's too high:

![Back link animation showing 1px vertical shift (Percy)](https://user-images.githubusercontent.com/415517/201368058-77ffc722-feaf-439b-8ced-85bfb9f18e43.gif)